sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -189,13 +189,11 @@ public partial class RequestConditions : Azure.MatchConditions
         public System.DateTimeOffset? IfModifiedSince { get { throw null; } set { } }
         public System.DateTimeOffset? IfUnmodifiedSince { get { throw null; } set { } }
     }
-    public partial class RequestContext
+    public partial class RequestContext : Azure.RequestOptions
     {
         public RequestContext() { }
         public System.Threading.CancellationToken CancellationToken { get { throw null; } set { } }
-        public Azure.ErrorOptions ErrorOptions { get { throw null; } set { } }
-        public void AddPolicy(Azure.Core.Pipeline.HttpPipelinePolicy policy, Azure.Core.HttpPipelinePosition position) { }
-        public static implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
+        public static new implicit operator Azure.RequestContext (Azure.ErrorOptions options) { throw null; }
```

Comment:
Why do we have the implicit cast but no .ctor?

I also wonder if we could hold off on the implicit cast for now because I worry so far our implicit casts have confused more customers than they've helped.

Reply:
It would be a breaking change to remove it.